src/styles/_vars.scss

```diff
@@ -16,6 +16,7 @@ $list-header-height: 32px;
 $list-width: 300px;
 
 // Media queries
+$phone: "max-width: 425px";
 $small: "max-width: 780px";
 $medium: "max-width: 1024px";
 $desktop: "min-width: 781px";
```

Comment:
At some point we should rename all of these to either device-based (phone, phablet, tablet, desktop, etc...) naming or size (tiny, small, medium, large)

Reply:
I'll call this one `$tiny` for now, since it fits with the other size-based names and them having a `max-width` value. `$desktop` has a `min-width` value and doesn't fit with the others.